application/template/NGSILD/goTemplate/function.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"fmt"
+	//. "nec-fogflow/common/ngsi"
+)
+
+type publishContextFunc func(ctxObj map[string]interface{})
+
+func fogfunction(ctxObj map[string]interface{}, publish publishContextFunc) error {
+	fmt.Println(ctxObj)
+	//==============Implemet losic ==============
```

Comment:
[minor] it should be implementation logic

Reply:
Done